### PR TITLE
[BOP-972] Add docs for GPU workloads

### PR DESCRIPTION
**Pull request by nwneisen**
https://mirantis.jira.com/browse/BOP-972

Adds documentation for enabling GPU in MKE4

Should be merged in after https://github.com/MirantisContainers/mke-operator/pull/40

### CODE REVIEW

**Comment by ry4nz** [comment]
Looks great!  Could you please also add a section talking about upgrade?

**Comment by nwneisen** [comment]
@ry4nz Thanks for the reminder!

**Comment by ry4nz** [comment]
@MagdaDziadosz please review

File: content/docs/migrate-from-MKE-3.md [hunk 1]
```diff
@@ -81,6 +81,9 @@ file:
 mkectl init --mke3-config </path/to/mke3-config.toml>
 ```
 
+{{< callout type="info" >}} If you are upgrading an MKE3 cluster with GPU enabled,
+you must perform the [GPU prerequisites](/docs/operations/gpu/#prerequisites) before upgrading {{< /callout >}}
```
Comment (MagdaDziadosz) [inline content/docs/migrate-from-MKE-3.md:85]:
```suggestion
{{< callout type="info" >}} To upgrade an MKE 3 cluster with GPU enabled,
ensure you complete the [GPU prerequisites](/docs/operations/gpu/#prerequisites) before
starting the upgrade process. {{< /callout >}}
```

File: content/docs/operations/gpu.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
+---
+title: GPU Workloads
+weight: 6
+---
+
+Mirantis Kubernetes Engine (MKE) supports running workloads on GPU nodes.
+Current support is limited to NVIDIA GPUs. MKE uses the NVIDIA GPU Operator
+to manage GPU resources on the cluster.
+
+To enable GPU support, MKE will install the NVIDIA GPU Operator on your cluster.
```
Comment (MagdaDziadosz) [inline content/docs/operations/gpu.md:10]:
```suggestion
To enable GPU support, MKE installs the NVIDIA GPU Operator on your cluster.
```

File: content/docs/operations/gpu.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+Before you can enable GPU support in MKE, you must install the [NVIDIA GPU
+toolkit](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/latest/install-guide.html)
+on each GPU enabled node in your cluster.
+
+## Configuration
+
+GPU support is disabled by default. To enable GPU support, you must configure
+the `gpu` section of the MKE configuration file.
+
```
Comment (MagdaDziadosz) [inline content/docs/operations/gpu.md:22]:
```suggestion
GPU support is disabled by default. To enable GPU support, configure
the `gpu` section of the MKE configuration file:

```

File: content/docs/operations/gpu.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+```yaml
+gpu:
+  enabled: true
+```
+
+This informs MKE to install the NVIDIA GPU Operator on your cluster.
```
Comment (MagdaDziadosz) [inline content/docs/operations/gpu.md:28]:
```suggestion
Configuration enables MKE to install the NVIDIA GPU Operator on your cluster.
```

**Review by MagdaDziadosz — CHANGES_REQUESTED** [review]

**Review by MagdaDziadosz — APPROVED** [review]

**Pull request merged**